src/core_plugins/kibana/public/visualize/listing/visualize_listing_table.js

```diff
@@ -0,0 +1,261 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { SortableProperties } from 'ui_framework/services';
+import { Pager } from 'ui/pager';
+import { NoVisualizationsPrompt } from './no_visualizations_prompt';
+
+import {
+  KuiPager,
+  KuiModalOverlay,
+  KuiConfirmModal,
+  KuiListingTableDeleteButton,
+  KuiListingTableCreateButton,
+  KuiListingTable,
+  KuiListingTableNoMatchesPrompt,
+  KuiListingTableLoadingPrompt
+} from 'ui_framework/components';
+
+export class VisualizeListingTable extends React.Component {
```

Comment:
I think the pattern I've been seeing more lately is to import `Component` from the `React` module instead.

```javascript
import React, {
  Component,
} from 'react';

export class VisualizeListingTable extends Component {
```